lib/Cro/HTTP/Router.pm6

```diff
@@ -947,15 +947,19 @@ module Cro::HTTP::Router {
 
         my %fallback = $mime-types // {};
         my $ext = $child eq '.' ?? $base.extension !! $child.IO.extension;
-        my $content-type = %mime{$ext} // %fallback{$ext} // 'application/octet-stream';
+
+        my sub get-mime($ext) {
```

Comment:
A sub is `my` by default in Perl 6 (though I note this isn't quite right on the sub below).